src/pages/admin/student/edit/actions/spec/studentFetchRequest.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from 'chai';
+import { } from 'mocha';
+import { } from 'core-js';
```

Comment:
Yes, we don't need import `chai', `mocha' or `core-js' because they are declared as global.

See [tsconfig.json](https://github.com/MasterLemon2016/LeanMood/blob/master/tsconfig.json#L11)

- We add karma-chai-sinon as global to avoid imports everywhere.
- I think core-js should be defined too, although it looks like it doesn't needed. But I think it's better to add it in `types` section to be more consistent.